# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core-deps-apple/.cargo-checksum.json
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-core-deps-apple/.cargo-checksum.json
--- a/third_party/rust/wgpu-core-deps-apple/.cargo-checksum.json	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-core-deps-apple/.cargo-checksum.json	Tue Oct 07 17:00:40 2025 +0000
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"7a8c769f2eb7ea6b5a24f430264b36c878413198a87a8b9faa65002d7be95f96","README.md":"729fdd16cb87ad318ed2bfc70593363aa324c804c825ffea7c42670dff681255","src/lib.rs":"54ef7b7d746c6b26b0ee65552cc969d69c6204d0d38b3678c40bffbfbc5a146c"},"package":null}
\ No newline at end of file
+{"files":{"Cargo.toml":"6c96bef3ae237c938710184702abb14d8ffb8b7d02a3af5d63a0a388879e7ecf","LICENSE.APACHE":"a6cba85bc92e0cff7a450b1d873c0eaa2e9fc96bf472df0247a26bec77bf3ff9","LICENSE.MIT":"dc0d97139e8205818c703741c7be7cb3b96888bd5917b8d6fc6133731e403c21","README.md":"729fdd16cb87ad318ed2bfc70593363aa324c804c825ffea7c42670dff681255","src/lib.rs":"54ef7b7d746c6b26b0ee65552cc969d69c6204d0d38b3678c40bffbfbc5a146c"},"package":null}
\ No newline at end of file